### DISCUSSION TITLE
Add a way to mention non-code contributors in generated release notes

### DISCUSSION THREAD

**Discussion by swissspidy** [community/community#36659] [Other Feature Feedback, Questions, & Ideas]
The automated release notes are very handy and reduce the tedious work of manually assembling a changelog. One feature I like is the "Contributors" section at the bottom of the release notes that shows the avatars of all people whose PRs made it into the release.

Some projects prefer not only highlighting code contributors to a particular release, but also people who contributed in another meaningful way (testers, reporters, PMs, etc.). It would be great if there was a way to manually add these people to the generated release notes, for example by listing them in `release.yml` or by providing an additional arg when using the `releases/generate-notes` API endpoint.

**Comment by ossanna16** [comment]
Hey there @swissspidy and thank you for the feedback! I'm glad you found our community 🙂

**Comment by mwithi** [comment]
I upvote the @swissspidy suggestion!

In our case we publish the release using a "release repository" that combines the work of different repositories, so for us is not possibile at the moment to have the list of contributors who actually contributed to the latters. We know its uncommon process though. It would be great to have a customizable list to be shown under the contributors' release section.

**Reply by mwithi** [reply]
**Update**: found (by chance) that if the relase note includes the contributors list at the end (even if in a `<details />` section) the avatars will appear in the related section, yay!

The release for you to check: https://github.com/informatici/openhospital/releases/tag/v1.12.0

![image](https://user-images.githubusercontent.com/2938553/218280548-cf9038dd-d5d5-4036-86e8-de7b3b41c7aa.png)

Not clear the order though... (it is not alphabetical, nor by rank, nor my typing order, etc..) it seems just random
For instance the first one is @xrmx 🤷‍♂️ and he's not a recent contributor...
